src/main/resources/application-sh.properties

```diff
@@ -1,11 +0,0 @@
-datarepo.gcs.allowReuseExistingBuckets=true
```

Comment:
I realized this doesn't actually do anything since I'm pointing to broad-jade-dev instead of broad-jade-sh